utils/data/configs.js

```diff
@@ -138,13 +139,18 @@ export default class ProjectConfigs {
         configs.ghosler.auth.user = user;
         if (configs.ghosler.url !== url) configs.ghosler.url = url;
 
-        if (ghostUrl === '' || ghostAdminKey === '') {
-            return {level: 'error', message: 'Ghost URL or Admin API Key is missing.'};
+        if (ghostUrl === '' || ghostAdminKey === '' || ghostAdminSecret === '') {
+            return {level: 'error', message: 'Ghost URL, Admin API Key or Secret is missing.'};
+        }
+
+        if (ghostAdminSecret.toString().length < 8) {
```

Comment:
Since we are generating the keys with a default length of 100, this isn't necessary but lets keep it anyways. Just in case...